tests/test_cloze.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
```

Comment:
Thanks for writing tests!

Could you remove the `#!/usr/bin/env python` and unset the executable bit? I think it may cause confusion for people who just want to run the test. If you want to execute this file you can always do it by running `python3 tests/test_cloze.py`.

Also side note: you shouldn't just put `python` in a shebang; you should always specify `python2` or `python3` unless it's compatible with both. There is a [PEP that I'm very familiar with](https://www.python.org/dev/peps/pep-0394/) that explains why :)

Reply:
Thanks for the link to the PEP. Great information.
The top line is removed and the exe bit is unset with https://github.com/kerrickstaley/genanki/pull/37/commits/2f188991b30d12d6659fb9e91afde220bdc12349